templates/404.twig

```diff
@@ -14,7 +14,7 @@
 				<div class="col-md-4">
 					{{ page_notfound_help }}
 					<form class="form search-form" action="{{ data_nav_bar.home_url }}">
-						<input class="form-control" placeholder="Search" name="s" aria-label="Search" type="text">
+						<input class="form-control" placeholder="{{ __( 'Search', 'planet4-master-theme' ) }}" name="s" aria-label="{{ __( 'Search', 'planet4-master-theme' ) }}" type="text">
```

Comment:
Please also escape the strings
`__( 'Search', 'planet4-master-theme' )|e('html_attr')`

Reply:
Ugh! Thanks, will keep that in mind. Fixed now.